src/entity/learning-outcome/learning-outcome.ts

```diff
@@ -31,14 +31,15 @@ export class LearningOutcome {
     return this._bloom;
   }
   set bloom(bloom: string) {
-    if (bloom && levels.includes(bloom.toLowerCase())) {
-      this._bloom = bloom.toLowerCase();
-    } else {
-      throw new EntityError(
-        LEARNING_OUTCOME_ERROR_MESSAGES.INVALID_BLOOM(bloom),
-        'bloom',
-      );
-    }
+    this._bloom = bloom.toLowerCase();
+    // This logic works; it just belongs in a service
```

Comment:
Are we going to uncomment this or delete it?

Reply:
^